CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ### Bugfixes
 
-* Lorem ipsum.
+* Fixes bug in encryption that could cause deadlocks/hangs and possibly
+  other bugs too. Fixes https://github.com/realm/realm-core/pull/2650
```

Comment:
@danielpovlsen It might be better to construct a proper test. My test is just a rewrite  of a test from Realm Java, so it is not on the level which is required for a core unit test.